### ISSUE TITLE
unnecessary function call (perhaps)

### ISSUE THREAD

**Issue by ImOver00** [openmultiplayer/web#552]
https://github.com/openmultiplayer/web/blob/3eac1c3f5f5801321bef2b6e533566af004e5440/server/main.go#L15
importing `autoload` will build it even tho its not used but it will execute the `init` function. Meanwhile the "init" function already does `godotenv.Load()`. so why are you importing autoload and doing that at the same time? Shouldn't you stick to ONLY one of these?

**Comment by Southclaws** [comment]
yeah it probably should be one or the other, can't remember why but I think it was load order. experiment with removing one and seeing if the app still works properly

**Comment by ImOver00** [comment]
thanks, will do soon
